docs/DeveloperGuide.md

```diff
@@ -100,15 +100,39 @@ _Written by: Author name_
 
 ### Overview for Transaction
 
-{Give a brief overview of the Transaction feature in Moolah Manager application.}
+{Give a brief overview of the Transaction features (i.e. purpose of each command) in Moolah Manager application.}
 
 _Written by: Author name_
 
 ### Implementation for Transaction
 
-{Provide the class diagram for Transaction} 
-
-_Written by: Author name_
+Each `Transaction` object in Moolah Manager represents a transaction record, which can be of `Income`
+or `Expense` type. Below is a simplified class diagram (note: methods have been omitted) containing the attributes
+within each transaction and how the transactions are associated with the `TransactionList`.
+
+<p align="center">
+    <img src="images/TransactionClassDiagram.png">
```

Comment:
The image does not render when I try to view the DG. I believe the proper markdown of an image in GitHub is 
`![Alt text](relative/path/to/img.jpg)`

Reply:
Nevermind ignore this, it doesn't render in the review page, but it does render outside after the merge.